src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2140,6 +2140,7 @@ static void updateAttrInNotifyCer
           caP->previousValue = new ContextAttribute();
```

Comment:
Not at all ...
Two new issues opened:
* https://github.com/telefonicaid/fiware-orion/issues/2588
* https://github.com/telefonicaid/fiware-orion/issues/2587

NTC (for this PR)